### PR TITLE
Add covariance for published odometry

### PR DESCRIPTION
**Pull request by lakid**
The node is modified to publish empirical Covariance that change according to if the robot is stationary or moving.

### CODE REVIEW

File: src/RosAria.cpp [hunk 1]
```diff
@@ -492,6 +522,20 @@ void RosAriaNode::publish()
   position.header.frame_id = frame_id_odom;
   position.child_frame_id = frame_id_base_link;
   position.header.stamp = ros::Time::now();
+
+  if(position.twist.twist.linear.x == 0 && 
+    position.twist.twist.linear.y == 0 && 
+    position.twist.twist.angular.z ==0 )
```
Comment (jasedit) [inline src/RosAria.cpp]: Minor spacing issue, otherwise should these be checking for some epsilon as opposed to zero?

File: src/RosAria.cpp [hunk 2]
```diff
@@ -39,6 +67,8 @@ class RosAriaNode
     RosAriaNode(ros::NodeHandle n);
     virtual ~RosAriaNode();
     
+
+
```
Comment (jasedit) [inline src/RosAria.cpp:69]: Boo additional whitespace.

File: src/RosAria.cpp (hunk 1 above)
Comment (lakid) [inline src/RosAria.cpp]: Well I wanted it to be 100\% certain it's stationary before giving lower covariance.
Later on we could change it if we decide to have covariance relative to speed

File: src/RosAria.cpp [hunk 3]
```diff
@@ -492,6 +520,20 @@ void RosAriaNode::publish()
   position.header.frame_id = frame_id_odom;
   position.child_frame_id = frame_id_base_link;
   position.header.stamp = ros::Time::now();
+
+  if(position.twist.twist.linear.x  == 0 && 
+     position.twist.twist.linear.y  == 0 && 
+     position.twist.twist.angular.z == 0)
```
Comment (lakid) [inline src/RosAria.cpp:526]: Wanted to be sure that the robot is 100\% stationary before sending good covariance values.

**Pull request merged**